pages/paper/[documentId]/[documentSlug]/index.tsx

```diff
@@ -93,6 +113,7 @@ const DocumentIndexPage: NextPage<Args> = ({
         errorCode={errorCode}
         metadata={documentMetadata}
         documentType={documentType}
+        twitterScore={twitterScore}
```

Comment:
I think `twitterScore` should live within `metadata`. We can chat about it if it isn't working